app/models/message.rb

```diff
@@ -67,11 +67,13 @@ def send_new_message_notification
   end
 
   def update_client_store
-    subscribed_user_channels = subscribed_user_channels()
-    unsubscribed_user_channels = Channel.users(User.staff) - subscribed_user_channels
+    sender_channel = Channel.user(sender)
+    subscribed_user_channels = subscribed_user_channels() - sender_channel
+    unsubscribed_user_channels = Channel.users(User.staff) - subscribed_user_channels - sender_channel
 
-    user = Api::V1::UserSerializer.new(self.sender)
+    user = Api::V1::UserSerializer.new(sender)
 
+    send_update self, user, 'read', sender_channel
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.